### PR TITLE
fix: Unable to load screen

### PR DESCRIPTION
**Pull request by rajsam003**
The type of this PR is: Bugfix

This PR resolves [CX-2405]

### Description

The unable to load screen was hidden behind the header on the profile screen for my collection and saved works. Its fixed in the PR. 

|Before|After|
|---|---|
|image|image|

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-2405]: https://artsyproduct.atlassian.net/browse/CX-2405?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Components/LoadFailureView.tsx [hunk 1]
```diff
@@ -7,9 +7,14 @@ import { ReloadIcon } from "../../palette/svgs/ReloadIcon"
...
   error?: Error
   onRetry?: () => void
+  displayInBottom?: boolean
 }
 
-export const LoadFailureView: React.FC<LoadFailureViewProps> = ({ error, onRetry }) => {
+export const LoadFailureView: React.FC<LoadFailureViewProps> = ({
+  error,
+  onRetry,
+  displayInBottom,
```
Comment (olerichter00) [inline src/app/Components/LoadFailureView.tsx:16]: I'm not sure if "display below" is grammatically correct. What do you think about renaming it to `placeAtBottom` or just pass `justifyContent` as a prop?

```suggestion
  /**
   * Places the content at the bottom of the screen.
   */
  placeAtBottom,
```

or 

```suggestion
  justifyContent,
```

**Review by olerichter00 — CHANGES_REQUESTED** [review]
Thanks a lot for the fix :)

File: src/app/Components/LoadFailureView.tsx (hunk 1 above)
Comment (rajsam003) [reply src/app/Components/LoadFailureView.tsx:16]: Passing `justifyContent` as a prop is a better idea. I will change it. Thanks

**Review by MrSltun — APPROVED** [review]
LGTM! Great work Sam! 👏 👏

**Review by olerichter00 — APPROVED** [review]
👏

**Pull request merged**